src/Classes/LinkRenderers/NavLinkRenderer.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace LaraZeus\Sky\Classes\LinkRenderers;
+
+use Illuminate\Database\Eloquent\Model;
+
+abstract class NavLinkRenderer
+{
+    public static string $renders;
+
+    public function __construct(
+        protected array $item
+    ) {
+    }
+
+    // TODO: something to control these classes for end-user?
+    public static string $activeClasses = 'border-b border-b-secondary-500 text-secondary-500';
+
+    public static string $defaultActiveClass = 'border-transparent';
+
+    abstract public function getModel(): ?Model;
+
+    abstract public function getLink(): ?string;
```

Comment:
Think this one was made nullable on accident 🤔 - technically the most "empty" thing we need out of this is an empty string. Don't think it actually needs to be able to be null?

Reply:
a use case come to mind, I did use it in on project but for the love of god cant find which one was...

the menu can have nested items, for a dropdown menu with sub nav, so maybe in this case the link can be null!